src/editor/contentState/pasteCtrl.js

```diff
@@ -44,8 +44,9 @@ const pasteCtrl = ContentState => {
     }
     event.preventDefault()
     const text = event.clipboardData.getData('text/plain')
-    let html = event.clipboardData.getData('text/html')
-
+    const html = event.clipboardData.getData('text/html')
+    console.log(text)
+    console.log(html)
```

Comment:
Maybe we need a `console.log` linter rule :smile:

Reply:
😆，if we add a `console.log` linter, it will always notice when debug.